tests/test-sources/plugins/utils/which-key.nix

```diff
@@ -8,46 +8,54 @@
       enable = true;
 
       # Testing for registrations
-      registrations."f" = {
-        prefix = "<leader>";
-        mode = [
-          "n"
-          "v"
-          "i"
-          "t"
-          "c"
-          "x"
-          "s"
-          "o"
-        ];
-        name = "Group Test";
-        f = "Label Test";
-        "1" = [
-          {
-            __raw = ''
-              function()
-                print("Raw Lua Code and List KeyMapping Test")
-              end
-            '';
-          }
-          "Raw Lua Code and List KeyMapping Test"
-        ];
-        "oo" = "Label Test 2";
-        "<tab>" = {
-          name = "Group in Group Test";
-          f = [
-            {
-              __raw = ''
-                function()
-                  vim.cmd("echo 'Raw Lua Code and List KeyMapping Test 2'")
-                end
-              '';
-            }
+      settings.spec = [
+        {
+          __unkeyed-1 = "f";
 
-            "Raw Lua Code and List KeyMapping Test 2"
+          prefix = "<leader>";
+          mode = [
+            "n"
+            "v"
+            "i"
+            "t"
+            "c"
+            "x"
+            "s"
+            "o"
           ];
-        };
-      };
+          name = "Group Test";
```

Comment:
Kinda confused by this one. In reading the documentation, we aren't expected to supply name anymore and the group has a group property.

Reply:
Looks like name is deprecated, so I'd recommend switching it to `group`

Reply:
Ah, it seems `group` should also be used instead of `desc` when registering a partial-binding (group)